lance/lance-rest-server/src/main/java/org/apache/gravitino/lance/service/rest/LanceTableOperations.java

```diff
@@ -116,24 +138,30 @@ public Response createTable(
   @ResponseMetered(name = "create-empty-table", absolute = true)
   public Response createEmptyTable(
       @PathParam("id") String tableId,
-      @QueryParam("mode") @DefaultValue("create") String mode, // create, exist_ok, overwrite
       @QueryParam("delimiter") @DefaultValue(NAMESPACE_DELIMITER_DEFAULT) String delimiter,
+      CreateEmptyTableRequest request,
       @Context HttpHeaders headers) {
     try {
-      // Extract table properties from header
-      MultivaluedMap<String, String> headersMap = headers.getRequestHeaders();
-      String tableLocation = headersMap.getFirst(LANCE_TABLE_LOCATION_HEADER);
-      String tableProperties = headersMap.getFirst(LANCE_TABLE_PROPERTIES_PREFIX_HEADER);
-
+      String tableLocation = request.getLocation();
       Map<String, String> props =
-          StringUtils.isBlank(tableProperties)
-              ? Map.of()
-              : JsonUtil.mapper().readValue(tableProperties, new TypeReference<>() {});
+          request.getProperties() == null
+              ? Maps.newHashMap()
+              : Maps.newHashMap(request.getProperties());
+      props.put("format", "lance");
```

Comment:
It's better to move this to `GravitinoLanceNamespaceWrapper` since it's a Gravitino-specific property.